_posts/2017-02-17-holacracy-and-purpose.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: Holacracy and purpose
+---
+
+{{page.title}}
+==============
+
+**In Holacracy, purpose is seen as the core concept around which everything else revolves. But what makes purpose so vital for a self-managing organization, and why should you care?**
+
+## A thought experiment
+
+Let's assume you are the managing director of a software company with three departments: Sales, Engineering, and Recruitment.
+
+One day, you get contrasting reports from two of your departments. Sales figures are better than ever before. Engineering, however, reports high work loads and not being able to deliver software on time.
+
+As a strategic direction, you decide engineering capacity should be increased. You ask the Recruitment department to take up a project to hire two new software engineers. As a first action, they should write up a job description and send it to you for final approval.
+
+## Same situation, different approach
+
+Now, let's do this same experiment for a self-managing organization running on Holacracy. I assume familiarity with Holacracy's basic concepts. Otherwise, consider watching [this animation video](https://www.youtube.com/watch?v=MUHfVoQUj54) or [this talk by Brian Robertson](https://www.youtube.com/watch?v=tJxfJGo-vkI) to get a quick overview.
+
+For the context of this article, we'll focus on the concept of _Lead Link_. In the Holacracy operating system, there is no such thing as a "managing director" or other "manager" or "leader" job titles. Instead, work is divided over circles and roles. Each circle has a purpose and a special role called Lead Link. One of the Lead Link's responsibilities is to ensure the circle is working towards its purpose.
+
+The top-level circle in an organization is called the _Anchor Circle_. The purpose of that circle is also the organization's overall purpose. The Lead Link of the Anchor Circle is accountable for discovering and expressing this purpose.
+
+Let's apply this to our software development company. Instead of being managing director, you now fill the Lead Link role for the company's Anchor Circle. You can still get the same reports: Sales is doing great, Engineering is struggling. Based on this information, you can also still make the same strategic decision to focus on increasing engineering capacity.
+
+So far so good. But what's different then?
+
+As managing director, you could manage everything from strategy down to projects and actions. As a Lead Link, your authority stops after the strategic level. In a self-managing organization, you can no longer _tell_ your Recruitment department (which would technically be called a Recruitment circle in Holacracy terms) to take up a project to hire two new engineers. You can _ask_, but ultimately the person responsible for recruitment is the one deciding if and when the project is taken up.
```

Comment:
Het verschil is hier nog niet helemaal duidelijk. Gaat het echt alleen maar om “vragen” in plaats van “vertellen”? Zou je niet zeggen dat je als lead link hier van een rep link wil horen dat de cirkel een spanning heeft? En dat je dan op basis van een purpose kiest om meer resources aan recruitment te besteden, omdat die manier van werken niet aansluit bij de missie?